app/lib/SecvisogramPage/View/FormEditorTab/shared/definitions/Lang.js

```diff
@@ -24,8 +24,8 @@ export default function Lang({
       label={label}
       description={description}
       deletable={deletable}
-      pattern="^[a-zA-Z]{2,3}(-.+)?$"
-      placeholder="^[a-zA-Z]{2,3}(-.+)?$"
+      pattern="^(([A-Za-z]{2,3}(-[A-Za-z]{3}(-[A-Za-z]{3}){0,2})?|[A-Za-z]{4,8})(-[A-Za-z]{4})?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-[A-WY-Za-wy-z0-9](-[A-Za-z0-9]{2,8})+)*(-[Xx](-[A-Za-z0-9]{1,8})+)?|[Xx](-[A-Za-z0-9]{1,8})+|[Ii]-[Dd][Ee][Ff][Aa][Uu][Ll][Tt]|[Ii]-[Mm][Ii][Nn][Gg][Oo])$"
+      placeholder="^[a-zA-Z]{2,3}(-.+)?$^(([A-Za-z]{2,3}(-[A-Za-z]{3}(-[A-Za-z]{3}){0,2})?|[A-Za-z]{4,8})(-[A-Za-z]{4})?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-[A-WY-Za-wy-z0-9](-[A-Za-z0-9]{2,8})+)*(-[Xx](-[A-Za-z0-9]{1,8})+)?|[Xx](-[A-Za-z0-9]{1,8})+|[Ii]-[Dd][Ee][Ff][Aa][Uu][Ll][Tt]|[Ii]-[Mm][Ii][Nn][Gg][Oo])$"
```

Comment:
Comment only and not directly related to this PR: 

As a developer I'd be able to grasp the former RegExp but I would feel inable to understand, what form the expected values should have. Furthermore in the UI, this string is most of the time displayed two times: As placeholder and below the field within the violated constraint message.

From a UX perspective I'd assume providing valid example as placeholder instead of Regex Patterns would be an improvement not only here but everywhere.

Reply:
Now tracked as #198.